cli/cluster.go

```diff
@@ -482,11 +483,20 @@ func runLogSinkAddSyslog(args *docopt.Args, client controller.Client) error {
 		return fmt.Errorf("Invalid syslog protocol: %s", u.Scheme)
 	}
 
+	var format ct.SyslogFormat
+	switch args.String["--format"] {
+	case "newline":
+		format = ct.SyslogFormatNewline
+	default:
+		format = ct.SyslogFormatRFC6587
```

Comment:
I think we should only use `ct.SyslogFormatRFC6587` if the flag is either `rfc6587` or empty, so if someone makes a typo like `--format newlin` this will error.